var/spack/repos/builtin/packages/wonton/package.py

```diff
@@ -113,10 +114,12 @@ def cmake_args(self):
         else:
             options.append('-DWONTON_ENABLE_Jali=OFF')
 
-        if '+flecsi' in self.spec:
-            options.append('-DWONTON_ENABLE_FleCSI=ON')
-        else:
-            options.append('-DWONTON_ENABLE_FleCSI=OFF')
+        # BROKEN DEPENDENCY!!!!!!
+        # if '+flecsi' in self.spec:
+        #     options.append('-DWONTON_ENABLE_FleCSI=ON')
+        # else:
+        #     options.append('-DWONTON_ENABLE_FleCSI=OFF')
+        options.append('-DWONTON_ENABLE_FleCSI=OFF')
```

Comment:
```suggestion
        options.append(define_from_variant('WONTON_ENABLE_FleCSI', 'flecsi')
```
If the variant is broken, add a `conflicts('+flecsi', msg='FleCSI is broken')` to the package rather than silently disabling it in the cmake logic.